src/main/java/gyro/google/dns/ResourceRecordSetResource.java

```diff
@@ -0,0 +1,240 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.dns;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import com.google.api.services.dns.Dns;
+import com.google.api.services.dns.model.Change;
+import com.google.api.services.dns.model.ResourceRecordSet;
+import com.google.api.services.dns.model.ResourceRecordSetsListResponse;
+import gyro.core.GyroException;
+import gyro.core.GyroUI;
+import gyro.core.Type;
+import gyro.core.resource.Resource;
+import gyro.core.resource.Updatable;
+import gyro.core.scope.State;
+import gyro.core.validation.Required;
+import gyro.google.Copyable;
+import gyro.google.GoogleResource;
+
+/**
+ * Creates a Resource Record Set.
+ *
+ * Example
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *     google::dns-resource-record-set managed-zone-example-public-www-cname
+ *         managed-zone: $(google::dns-managed-zone managed-zone-example-public)
+ *
+ *         name: "www.p.example.com."
+ *         type: "CNAME"
+ *         rrdatas: [ "fe.p.example.com." ]
+ *     end
+ *
+ *     google::dns-resource-record-set managed-zone-example-public-fe-a
+ *         managed-zone: $(google::dns-managed-zone managed-zone-example-public)
+ *
+ *         name: "fe.p.example.com."
+ *         type: "A"
+ *         rrdatas: [ "192.168.1.156", "192.168.1.158" ]
+ *     end
+ */
+@Type("dns-resource-record-set")
+public class ResourceRecordSetResource extends GoogleResource implements Copyable<ResourceRecordSet> {
+
+    private ManagedZoneResource managedZone;
+
+    private String name;
+
+    private List<String> rrdatas;
+
+    private List<String> signatureRrdatas;
+
+    private Integer ttl;
+
+    /**
+     * The identifier of a supported record type. See the list of Supported DNS record types.
+     */
+    private String type;
+
+    /**
+     * The managed zone this resource should be bound to.
+     *
+     * @resource gyro.google.dns.ManagedZoneResource
+     */
+    @Required
+    public ManagedZoneResource getManagedZone() {
+        return managedZone;
+    }
+
+    public void setManagedZone(ManagedZoneResource managedZone) {
+        this.managedZone = managedZone;
+    }
+
+    /**
+     * For example, www.example.com.
+     */
+    @Updatable
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    /**
+     * As defined in RFC 1035 (section 5) and RFC 1034 (section 3.6.1) -- see examples.
+     */
+    @Updatable
+    public List<String> getRrdatas() {
+        if (rrdatas == null) {
+            rrdatas = new ArrayList<>();
+        }
+        return rrdatas;
+    }
+
+    public void setRrdatas(List<String> rrdatas) {
+        this.rrdatas = rrdatas;
+    }
+
+    /**
+     * As defined in RFC 4034 (section 3.2).
```

Comment:
Is there another way to convey what this field is in a concise way instead of relying only on the RFC doc?